annif/backend/http.py

```diff
@@ -11,6 +11,33 @@
 class HTTPBackend(backend.AnnifBackend):
     name = "http"
 
+    @property
+    def is_trained(self):
+        return self._get_project_info('is_trained')
+
+    @property
+    def modification_time(self):
+        return self._get_project_info('modification_time')
+
+    def _get_project_info(self, key):
+        params = self._get_backend_params(None)
+        try:
+            req = requests.get(params['endpoint'].replace('/suggest', ''))
+            req.raise_for_status()
+        except requests.exceptions.RequestException as err:
+            self.warning("HTTP request failed: {}".format(err))
```

Comment:
Should this be changed into raising an OperationFailedException, like the `maui backend does? I know that it's not used in the `suggest` method of the `http` backend, but perhaps it would be better to use it also there.

Reply:
I changed the `_get_project_info()` of http and Maui backends to raise the `OperationFailedException` instead of only warn in case http or json failures. The raised exception is then handled in `_get_info()` of project module (to allow `list-projects` command to continue after a failing http project).